index.html

```diff
@@ -178,5 +178,58 @@ <h4 class="skill-title">Skills</h4>
       </div>
     </div>
   </section>
+  <section class="section-contact-form">
+    <div class="contact-form-container grid gen-margin">
+      <h3 class="contact-form-primary">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      <form action="" class="contact-form">
+        <input type="text" class="form-input" placeholder="Full name">
+        <input type="email" class="form-input" placeholder="Email address">
+        <textarea name="message" class="form-input" placeholder="Write me something..."></textarea>
```

Comment:
- [ ] For your message input, It should have a  `maxlength="500"` attribute.
- This is also in reference to the project requirements 👍 
- All your input fields should also  have the `required` attribute to activate the validation on submit.